Cargo.toml

```diff
@@ -17,8 +17,13 @@ appveyor = { repository = "saschagrunert/indextree", branch = "master", service
 
 [features]
 deser = [ "serde", "serde_derive" ]
+par_iter = ["rayon"]
```

Comment:
Can you please add a build including this feature in the appveyor and travis config?